Source/Shaders/PostProcessFilters/FXAA3_11.glsl

```diff
@@ -0,0 +1,2074 @@
+/**
```

Comment:
This file is really long.  Our build process will remove comments from release builds, but it will not remove all the versions of this shader that are pulled out at runtime by the preprocessor.  I hate to bloat Cesium with that extra code and potentially slow down shader compile times.

Can we either
* Add the preprocessor directives to this file and verify/change our build process to generate the smaller shader, or
* Generate the smaller shader by hand.

In either case, please document how to upgrade this.  A comment at the top of the file should be OK.